test/tests/functional/pbs_ralter.py

```diff
@@ -1184,67 +1195,67 @@ def test_alter_degraded_resv_mom_down(self):
                                  alter_e=True, whichMessage=3, interactive=2,
                                  sequence=2)
         self.mom.start()
-
-    def test_sched_cycle_kicks_on_resv_end(self):
-        """
-        This test checks whether the sched cycle gets kicked
-        when the advance reservation ends
-        """
-        duration = 10
-        offset = 5
-
-        rid, _, _ = self.submit_and_confirm_reservation(
-            offset, duration, select="1:ncpus=8")
-        attr = {'Resource_List.walltime': '00:00:20'}
-        j = Job(TEST_USER)
-        j.set_sleep_time(10)
-        j.set_attributes(attr)
-        jid = self.server.submit(j)
-        self.server.expect(JOB, {'job_state': 'Q'},
-                           id=jid, max_attempts=5, interval=0.5)
-        self.scheduler.log_match(
-            jid + ";Job would conflict with reservation or top job",
-            n='ALL',  max_attempts=30, interval=0.5)
-        self.server.log_match(
-            rid.split('.')[0] + ";deleted at request of pbs_server",
-            id=rid.split('.')[0], n='ALL', max_attempts=30, interval=1)
-        self.server.log_match(jid + ";Job Run", id=jid,
-                              n='ALL', max_attempts=30, interval=0.5)
+        # test same for node offline case
+        attrs1 = {'reserve_state': (MATCH_RE, 'RESV_CONFIRMED|2')}
+        self.server.expect(RESV, attrs1, id=rid1)
+        self.server.expect(RESV, attrs1, id=rid2)
+        resv_node1 = self.server.status(RESV, {'resv_nodes'}, id=rid1)[0][
+            'resv_nodes'].split(':')[0].split('(')[1]
+        resv_node2 = self.server.status(RESV, {'resv_nodes'}, id=rid2)[0][
+            'resv_nodes'].split(':')[0].split('(')[1]
```

Comment:
{'resv_nodes'} is a malformed dictionary.  The key resv_nodes does not have a value.  expect() can take a string, so just remove the {}

Reply:
Done